test/workbox-webpack-plugin/node/v4/generate-sw.js

```diff
@@ -6,24 +6,35 @@
   https://opensource.org/licenses/MIT.
 */
 
+// workbox-webpack-plugin needs to do require('webpack'), and in order to test
+// against multiple webpack versions, we need that to resolve to whatever the
+// correct webpack is for this test.
```

Comment:
Maybe add a link somewhere to [your post about this](https://jeffy.info/2020/10/01/testing-multiple-webpack-versions.html), so anyone looking at this in the future can understand why this was done?

Reply:
Sure.